src/config.js

```diff
@@ -0,0 +1,66 @@
+const chalk = require('chalk');
+const { join } = require('path');
+
+const getServePath = (dir, path) => {
+  if (typeof path !== 'string' || typeof dir !== 'string') {
+    return { path: undefined };
+  }
+
+  const resolvedPath = join(dir, path);
```

Comment:
Added a test case for this https://github.com/netlify-labs/netlify-plugin-lighthouse/pull/45/commits/d892f583fec5988434c87ea916bda1f38cf67493#diff-717adf98883419d80e527cbc26159056R128

`path.resolve('bar', '/foo')` returns `/foo` while `path.join('bar', '/foo')` returns `bar/foo`.

Since we're expecting the result to always be under the base directory, it is better to use `join`.